hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -166,10 +166,10 @@
                cov: true
              }
              {
-               name: rv_plic_fpv
-               fusesoc_core: lowrisc:fpv:rv_plic_fpv
+               name: top_earlgrey_rv_plic_fpv
```

Comment:
it looks like we need to change this name field back to `rv_plic_fpv` since this is used as the DUT name in the FPV run script.